### DISCUSSION TITLE
Any plans to add `isUndefined()`?

### DISCUSSION THREAD

**Discussion by RobinBobin** [radashi-org/radashi#303] [Q&A] [answered]
Hi!
There are `isString()`, `isNumber()`, etc. Any plans of adding `isUndefined()`?
Thanks.

**Comment by aleclarson** [comment]
Sure. PR welcome 👍

**Comment by RobinBobin** [comment, marked as answer]
:pray:  done
https://github.com/radashi-org/radashi/pull/305
